### PR TITLE
NXCT-134: Hide unavailble features

### PR DESCRIPTION
**Pull request by egiuly-nuxeo**
I hide:
- postgres (=> there is only dbs)
- kafka (=> there is only chronicle)
- stream and redis (=> there is only inmemory)
- minio and s3 (=> there is only fs)
- studio-only plan (=> there is only default plan) 

Also I force:
- gp2 for backend storage class
- shared for binaries storage class

### CODE REVIEW

File: nuxeo-dev-apb/apb.yml [hunk 1]
```diff
@@ -109,81 +109,97 @@ plans:
...
         title: Binaries storage type
         display_group: Backend storages
 
       # TODO: K/V store type
 
       - name: backend_pv_storage_class
-        type: string
         title: Storage class for backend persistence volumes
-        default: gp2
+        # TODO: Make other backend storage classes work before showing this field as a free text field.
```
Comment (dmetzler) [inline nuxeo-dev-apb/apb.yml]: Storage class must be kept free-from as they are declared at the Openshift level and as they are not managed by this APB

File: nuxeo-dev-apb/apb.yml [hunk 2]
```diff
@@ -109,81 +109,97 @@ plans:
...
+        # default: gp2
+        type: enum
+        enum: ['gp2']
         display_group: Backend storages
 
       - name: binaries_pv_storage_class
-        type: string
         title: Storage class for backend persistence volumes (if fs used)
-        default: shared
+        # TODO: Make other binaries storage classes work before showing this field as a free text field.
```
Comment (dmetzler) [inline nuxeo-dev-apb/apb.yml]: Storage class must be kept free-from as they are declared at the Openshift level and as they are not managed by this APB

File: nuxeo-dev-apb/apb.yml (hunk 1 above)
Comment (egiuly-nuxeo) [reply nuxeo-dev-apb/apb.yml]: ok, fixed

File: nuxeo-dev-apb/apb.yml (hunk 2 above)
Comment (egiuly-nuxeo) [reply nuxeo-dev-apb/apb.yml]: ok, fixed

**Comment by egiuly-nuxeo** [comment]
@dmetzler 
`apb prepare` => https://github.com/nuxeo/nuxeo-openshift/pull/11/commits/19552623ad2fc221deea3696fa631ba035dd29ae

**Pull request merged**